pipelined/bevy_pbr2/src/render/light.rs

```diff
@@ -550,7 +580,7 @@ pub fn prepare_lights(
                 let view_light_entity = commands
                     .spawn()
                     .insert_bundle((
-                        ViewLight {
+                        ShadowView {
```

Comment:
It seems like we should just not create "shadow view" Components and gpu resource at all for lights with shadows disabled. And in a world where views support many point lights (clustered-forward, especially with storage buffers), we probably shouldn't be allocating texture space for every theoretically possible shadow producer anyway. However with the current impl a 3D texture of depth 10 (or 1) isn't something to be worried about.

Reply:
Also by adding `ShadowView` / `RenderPhase<Shadow>` for lights with disabled shadows,  won't we start a bunch of "empty" shadow passes? That probably introduces some amount of overhead.

Reply:
Putting the cubemap for loop in a `if light.shadows_enabled` block seems to do the trick.